sdk/src/types/block/mod.rs

```diff
@@ -12,6 +12,8 @@ mod issuer_id;
 
 /// A module that provides types and syntactic validations of addresses.
 pub mod address;
+/// A module that provides functionality for capabilities.
```

Comment:
```suggestion
/// A module that provides functionality for transaction capabilities.
```

Reply:
These are not transaction capabilities

Reply:
what are they? the PR is called like this.

Reply:
Just capabilities, they are generic. Address capabilities also use these.